shared/teams/teamname.js

```diff
@@ -0,0 +1,75 @@
+// @flow
+import * as I from 'immutable'
```

Comment:
we should move these types of helpers into constants/team or into a folder there vs w/ the components

Reply:
Done, moved to constants